waiter/src/waiter/util/http_utils.clj

```diff
@@ -75,3 +75,8 @@
       (let [new-user-agent-field (HttpField. HttpHeader/USER_AGENT (str user-agent))]
         (.setUserAgentField client new-user-agent-field)))
     client))
+
+(defn determine-backend-protocol-version
```

Comment:
Can we add a comment about the expectations of this function?

Reply:
Improved documentation with a TODO.